dirty_cat/test/test_docstrings.py

```diff
@@ -0,0 +1,172 @@
+import re
+from inspect import signature
+import pytest
+import inspect
```

Comment:
It's a minor nitpick, but it's nice to order the imports in a way. Right now we have two imports of inspect, but not consecutive. The typical ordering that I know is: std lib, classic packages, less classic packages, local import. And inside a given category, alphabetic ordering.

Reply:
FYI, `isort`, set in the `pre-commit` setup should sort the import canonically.

If it has not, you can run:

```bash
pre-commit run --from-ref=upstream/master --to-ref=prepare_docstring_meta_issue
```

Reply:
Thanks for pointing this out and thanks to @jjerphan for the pre-commit tricks. It is now fixed.

Reply:
I think there was a misunderstanding here: from what I understand, your description explains why we use `re`, but I think what Gaël meant was to clarify what this file (`test_docstrings`) does.

I'm thinking of something along the lines of *"This test suite ensures the docstrings of functions and classes in dirty_cat are formatted according to numpydoc specifications"*.